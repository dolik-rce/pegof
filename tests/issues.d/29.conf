input issues.d/29.peg
optimize all
header never
