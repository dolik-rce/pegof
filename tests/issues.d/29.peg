f <- d / d / 'x' d
d <- [\u0032-\u0033]
