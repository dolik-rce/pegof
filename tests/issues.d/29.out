f <-
    [23]
    / "x" [23]
